src/rootUtils.h

```diff
@@ -16,13 +17,54 @@
 #include <vector>
 
 namespace podio::root_utils {
+/**
+ * The name of the meta data tree in podio ROOT files. This tree mainly stores
+ * meta data that is necessary for ROOT based I/O.
+ */
+constexpr static auto metaTreeName = "podio_metadata";
```

Comment:
not sure such information is ROOT specific. It is part of the file schema and could maybe go elsewhere